### PR TITLE
vimPlugins: add jedi-vim

### PR DESCRIPTION
**Pull request by schmittlauch**
This is the master branch equivalent of  #49589, as the vimPlugins structure differs between master and release right now.

running the `updated.py` as described in the default.nix also updated
other vim plugins. If that's not wanted, I can reduce this changeset to
the jedi-vim related changes only.

As this is a python3 plugin it'd be nice to check for python3 support in
the used vim, but apparently nobody else does this.

###### Things done

- [ ] Tested using sandboxing ([nix.useSandbox](http://nixos.org/nixos/manual/options.html#opt-nix.useSandbox) on NixOS, or option `sandbox` in [`nix.conf`](http://nixos.org/nix/manual/#sec-conf-file) on non-NixOS)
- Built on platform(s)
   - [x] NixOS
   - [ ] macOS
   - [ ] other Linux distributions
- [ ] Tested via one or more NixOS test(s) if existing and applicable for the change (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
- [ ] Tested compilation of all pkgs that depend on this change using `nix-shell -p nox --run "nox-review wip"`
- [ ] Tested execution of all binary files (usually in `./result/bin/`)
- [ ] Determined the impact on package closure size (by running `nix path-info -S` before and after)
- [ ] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/.github/CONTRIBUTING.md).

---

### CODE REVIEW

**Comment by schmittlauch** [comment]
ping @jagajaga

**Review by andir — CHANGES_REQUESTED** [review]
What I'd like to see here is that the bump of the other plugins in a commit separated from the addition of jedi-vim.

**Comment by schmittlauch** [comment]
But the version bumps can still be included, just separated to another commit?

**Comment by andir** [comment]
Yes, They will either happen now or someone will do them later... I do not see a test framework to verify that they actually work..

**Comment by schmittlauch** [comment]
@andir splitting is done.

**Review by andir — CHANGES_REQUESTED** [review]
Looks good just some nitpicks.

File: pkgs/misc/vim-plugins/overrides.nix [hunk 1]
```diff
@@ -334,4 +334,15 @@ with generated;
     };
   });
 
+  jedi-vim = jedi-vim.overrideAttrs(old: {
+    # checking for python3 support in vim would be neat, too, but nobody else seems to care
+    buildInputs = [ python3Packages.jedi ];
+    meta = {
+      description = "code-completion for python using python-jedi";
+      license = stdenv.lib.licenses.mit;
+  };
```
[the remark this answers is not in the document]
Comment (schmittlauch) [reply pkgs/misc/vim-plugins/overrides.nix]: fixed

File: pkgs/misc/vim-plugins/overrides.nix [hunk 2]
```diff
@@ -334,4 +334,15 @@ with generated;
...
+  });
+
```
[the remark this answers is not in the document]
Comment (schmittlauch) [reply pkgs/misc/vim-plugins/overrides.nix]: fixed

**Review by andir — APPROVED** [review]

**Pull request merged**